oauthenticator/cilogon.py

```diff
@@ -101,6 +104,15 @@ def _validate_scope(self, proposal):
             Contact help@cilogon.org to request a custom skin.
         """,
     )
+    claim = Unicode(
+        config=True,
+        default_value = "sub", # Username is "sub" claim, or what specified in jupytherhub_config.py
```

Comment:
Can we change the default to "email"? The sub claim produces an invalid username because it's a URL, which won't work.

Reply:
I can certainly do that, however that's changing the current behavior in a backward incompatible way. Do you want me to do that?

Reply:
Well, the last version that worked used email, when we were using OAuth1. The OAuth2 update changed the key for the name to `sub` in the latest release. That has never worked due to the issues reported in #133, where it's producing an invalid username. So I don't think it's a breaking change, since we can't break something that doesn't work yet.